app/invocation/invocation_cache_card.tsx

```diff
@@ -213,3 +272,69 @@ export default class CacheCardComponent extends React.Component<Props> {
     );
   }
 }
+
+let cap = 5;
```

Comment:
nit: since this is a top-level declaration, rename this to be more clear? or move it to renderBreakdown

Reply:
Done